index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
> Broberg wanted to keep the site simple, with minimal dependencies on external libs.

Why, if I may ask? It's not like the smooth scrolling lib is thousands of lines of code ;) It does one thing and does it well.